src/site/stages/build/drupal/graphql/nodeStepByStep.graphql.js

```diff
@@ -0,0 +1,54 @@
+const entityElementsFromPages = require('./entityElementsForPages.graphql');
+
+const fragment = `
+fragment nodeStepByStep on NodeStepByStep {
+  ${entityElementsFromPages}
+  entityBundle
+
+  title
+  fieldIntroTextLimitedHtml {
+    processed
+  }
+  fieldAlertSingle {
+    entity {
+      ... alertSingle
+    }
+  }
+  fieldButtons {
+    entity {
+      ... button
+    }
+  }
+  fieldSteps {
+    entity {
+      ... on ParagraphStepByStep {
+        fieldStep {
+          entity {
+            ... on ParagraphStep {
+              fieldWysiwyg {
+                processed
+              }
```

Comment:
Not positive  but I think you can make use if a predefined fragment here too
```suggestion
            ... on ParagraphStep {
              fieldWysiwyg {
               ... wysiwyg
              }
```

Reply:
I got drupal errors trying to implement this:

![image](https://user-images.githubusercontent.com/12773166/92957347-9e98c200-f425-11ea-9e39-f259b26e2032.png)